perf_test/batched/KokkosBatched_Test_Gemm_Host.hpp

```diff
@@ -276,95 +276,74 @@ namespace KokkosBatched {
           {
             double tavg = 0, tmin = tmax;
 
-            MKL_INT blksize[1] = { BlkSize };
-            MKL_INT lda[1] = { a.stride_1() };
-            MKL_INT ldb[1] = { b.stride_1() };
-            MKL_INT ldc[1] = { c.stride_1() };
+            double done(1.0);
+            std::complex<double> zone(1.0);
```

Comment:
Hopefully this perf_test path won't be compiled for CUDA ?

Reply:
@srajama1 I looked a bit at this part of the test, it is all host code so only the host compiler should be compiling this, and zone is passed to the complex version of some mkl routine so using a `Kokkos::complex` would break the mkl interface. In addition, intel doesn't play with cuda, so we shouldn't run into a situation where this case could cause problems, good catch to check that spot though :)